src/preload.py

```diff
@@ -78,22 +81,24 @@ def __init__(self):
         self.counter = {}
 
     @staticmethod
-    def key(func_name, *args):
-        return " ".join(str(e) for e in (func_name,) + args)
+    def key(func_name, *args, **kwargs):
+        return " ".join(
+            str(e) for e in (func_name,) + args + tuple(kwargs.values())
+        )
 
-    def clear(self, key, *args):
+    def clear(self, key):
```

Comment:
Fix of a harmless bug, finally addressing Alexis' request in this older PR comment: https://github.com/balena-io-modules/balena-preload/pull/220#discussion_r455057117